src/collective/fingerpointing/browser/logview.py

```diff
@@ -32,10 +35,35 @@ def tail(self, f, n=100):
         return lines[-n:]
 
     @property
+    def _get_audit_log_files(self):
+        """Return an ordered list of log files to be processed. We need
+        to process the current audit log and all of its backups.
+        """
+        # get all audit log backups in the log directory
+        logdir = os.path.dirname(LOGFILE)
+        logfiles = os.listdir(logdir)
+        # filter the list and sort it in reverse order
+        # as we will need to process newer backup files first
+        logfiles = sorted([
+            f for f in logfiles
+            if fnmatch(f, 'audit.log.*') and '.lock' not in f
```

Comment:
instead of fnmatch I would use [glob module](https://docs.python.org/2/library/glob.html)